_includes/header.html

```diff
@@ -11,7 +11,7 @@
       <a class="mdl-navigation__link" href="{{ site.baseurl }}{% link projects.md %}">Projects</a>
       <a class="mdl-navigation__link" href="{{ site.baseurl }}{% link events.md %}">Events</a>
       <a class="mdl-navigation__link" href="">Membership</a>
-      <a class="mdl-navigation__link" href="">Governance</a>
+      <a class="mdl-navigation__link" href="/governance/">Governance</a>
```

Comment:
Give `{{ site.baseurl}}{% link governance/README.md %}` a try.
The nice part of using Jekyll `link` is that is will remain consistent even if the permalink changes.